airbyte-cdk/bulk/toolkits/extract-jdbc/src/main/kotlin/io/airbyte/cdk/read/SelectQuerier.kt

```diff
@@ -24,12 +24,23 @@ interface SelectQuerier {
     data class Parameters(
         /** When set, the [ObjectNode] in the [Result] is reused; take care with this! */
         val reuseResultObject: Boolean = false,
-        /** JDBC fetchSize value. */
-        val fetchSize: Int? = null,
-    )
+        /** JDBC [PreparedStatement] fetchSize value. */
+        val statementFetchSize: Int? = null,
+        /** JDBC [ResultSet] fetchSize value. */
+        val resultSetFetchSize: Int? = null,
+    ) {
+        constructor() : this(false, null, null)
+        constructor(
+            reuseResultObject: Boolean,
+            fetchSize: Int?
+        ) : this(reuseResultObject, fetchSize, fetchSize)
```

Comment:
this constructor maintains backward compatibility

Reply:
nit: might be clearer to use the named parameters